lib/trie/leaf/copy.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package leaf
```

Comment:
Stutter here as well `leaf.Leaf`.

Reply:
Done in https://github.com/ChainSafe/gossamer/pull/2077/commits/30e2318a9ddfca712a319b3aecce07beaf252d85

`branch`, `leaf`, `encode` and `decode` are merged in `lib/trie/node`